scripts/deps-ppa/static_z3.sh

```diff
@@ -78,6 +78,22 @@ Homepage: https://github.com/Z3Prover/z3
 Vcs-Git: git://github.com/Z3Prover/z3.git
 Vcs-Browser: https://github.com/Z3Prover/z3
 
+Package: z3-static
+Architecture: any
+Breaks: z3
+Replaces: z3
+Depends: ${misc:Depends}, ${shlibs:Depends}
```

Comment:
I think these should be escaped like the ones below. Otherwise Bash will interpret them as variables from the current script and perform substitution.
```suggestion
Depends: \${misc:Depends}, \${shlibs:Depends}
```

Reply:
By the way, these all seem to be just static files without any need for substitutions. Is there any reason why they're not separate files in the repo? The fact that they're long and cannot be indented just makes the whole script seem more complex than it actually is.

Reply:
Also no real reason other than the release_ppa.sh script also doing it like this. But you're right about escaping in any case.